.mergify.yml

```diff
@@ -29,19 +32,34 @@ pull_request_rules:
       merge:
         method: rebase
         strict: smart
+      label:
+        remove:
+        - "ready to merge"
   - name: label work in progress PRs
     conditions:
       - label != "work in progress"
-      - title ~= '^WIP\b'
+      - title ~=^WIP\b
```

Comment:
I had to change this, since mergify no longer likes the previous format. It wouldn't trigger these actions as the regex apparently no longer matched.